scripts/release/asset_publish.py

```diff
@@ -402,6 +405,49 @@ def create_asset(
         failure_list.append(asset)
 
 
+def update_asset_metadata(asset: AssetConfig, registry_name: str):
+    """Update the mutable metadata of asset."""
+    if asset.type == assets.AssetType.MODEL:
+        model_name = asset.name
+        model_version = asset.version
+        spec_path = asset.spec_with_path
+        model_config = asset.extra_config_as_object()
+
+        tags, description = None, None
+
+        # get tags to update from model spec file
+        try:
+            with open(spec_path) as f:
+                model_spec = YAML().load(f)
+                if "tags" in model_spec:
+                    tags = {"replace": model_spec["tags"]}
+        except Exception as e:
+            logger.log_error(f"Failed to get tags for model {model_name}: {e}")
+
+        # get updated description for model
+        try:
+            model_description_file_path = Path(spec_path).parent / model_config.description
+            logger.print(f"model_description_file_path {model_description_file_path}")
+            if os.path.exists(model_description_file_path):
+                with open(model_description_file_path) as f:
+                    description = f.read()
```

Comment:
Retrieval of the contents of the description file should be added to the ModelConfig class.

Reply:
Done